CIP-XXXX/README.md

```diff
@@ -0,0 +1,312 @@
+   </td>
+  </tr>
+</table>
+
+### PriceMap(Tag +2)
+
+This tag represents a single price sample at a specific point in time.
+
+Data inside this tag is stored as a CBOR Mapping, the keys defined as positive integers and the values being defined on a per-case basis. The keys are defined here as enums, to give users of the standard a common understanding of what data is stored where. Each field in the mapping is optional.
+
+*Keys-value mappings*
+
+<table>
+  <tr>
+   <td><strong>Key</strong>
+   </td>
+   <td><strong>Value Data Type</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>0
+   </td>
+   <td>Integer
+   </td>
+   <td>Price, or how many Quoted Currency is received per Base Currency spent
+   </td>
+  </tr>
+  <tr>
+   <td>1
+   </td>
+   <td>Integer
+   </td>
+   <td>POSIX Timestamp related to the time that the data was aggregated. 
+   </td>
+  </tr>
+  <tr>
+   <td>2
+   </td>
+   <td>Integer
+   </td>
+   <td>POSIX Timestamp related to the time that the oracle deems the data stale and/or expired.
+   </td>
+  </tr>
+  <tr>
+   <td>3
+   </td>
+   <td>Integer
+   </td>
+   <td>Decimals of Price
+   </td>
+  </tr>
+  <tr>
+   <td>4
+   </td>
+   <td>Integer
+   </td>
+   <td>ID of Base Currency
+   </td>
+  </tr>
+  <tr>
+   <td>5
+   </td>
+   <td>Integer
+   </td>
+   <td>ID of Quoted Currency
+   </td>
+  </tr>
+  <tr>
+   <td>6
+   </td>
+   <td>String
+   </td>
+   <td>Symbol of Base Currency
+   </td>
+  </tr>
+  <tr>
+   <td>7
+   </td>
+   <td>String
+   </td>
+   <td>Symbol of Quoted Currency
+   </td>
+  </tr>
+  <tr>
+   <td>8
+   </td>
+   <td>String
+   </td>
+   <td>Name of Base Currency
+   </td>
+  </tr>
+  <tr>
+   <td>9
+   </td>
+   <td>String
+   </td>
+   <td>Name of Quoted Currency
+   </td>
+  </tr>
+</table>
+
+
+
+### KYCList(Tag +3)
+
+This tag represents 0-X Cardano addresses and what country the controlling user is KYC’ed as being a citizen or tax resident of.
+
+Data inside this tag is stored as a CBOR Mapping, the keys defined as bytes representing Cardano addresses and the values being defined as a list of country codes as strings. Multiple country codes are to be interpreted as the address potentially belonging to any one of those.
+
+
+```
+cardano_public_key        = tstr
+nationality               = tstr
+whitelisted               = uint
+```
+
+<table>
+  <tr>
+   <td>
+
+<strong>Key</strong>
+   </td>
+   <td><strong>Value Data Type</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>0
+   </td>
+   <td>String
+   </td>
+   <td>Cardano Address Public Key
+   </td>
+  </tr>
+  <tr>
+   <td>1
+   </td>
+   <td>String
+   </td>
+   <td>Nationality
+   </td>
+  </tr>
+  <tr>
+   <td>2
+   </td>
+   <td>Integer (boolean)
+   </td>
+   <td>Address is whitelisted for sale
+   </td>
+  </tr>
+</table>
+
+## Rationale
+
+This CIP is ready to be made active; since it is a process standard, it requires no implementation.
```

Comment:
Actually, it does require projects adopting the standard. This is usually a good requirement for the **Path to Active.** section: have several projects adopting the standard specified in the document. If there are already some known projects that intend to, they can also explicitly be listed here. 

Note also that this covers the **Path to Active** section, but not the **Rationale** section. Rationale is about providing justifications for the various design decisions made in the proposed document. For example, it can show how the solution compares to the state of the art on other blockchain ecosystem, or explain the rationale behind the choice of a particular technology. The rationale may also explain why some solutions that could have been considered were rejected.

Reply:
I think that is a fair expectation. As has been mentioned before Charli3 are currently implementing this standard in publicly accessible, partially Catalyst-funded community feeds, which we hope can help wider adoption of the standard.